### DISCUSSION TITLE
A step-by-step guide

### DISCUSSION THREAD

**Discussion by pesik82** [Azure/azqr#86] [Q&A] [answered]
Can you please help me with an explanation on the Azure environment ? What am i supposed to configure?
I've opened the solution in VSC and installed the azqr .
If I'm trying to run the ./azqr scan by itself , it just open an empty tab.

**Comment by cmendible** [comment, marked as answer]
Hi, did you read: https://github.com/Azure/azqr#usage

Can you share the command you are running and share the output?
